### PR TITLE
OpenSource 2018 Post [WIP]

### PR DESCRIPTION
**Pull request by rohinb2**
Feel free to critique and change whatever! Open to all suggestions this is only first draft, by editing over the weekend we can hope to post this next week.

Also, the section on Sponsors at the end might seem a bit out of place but I think it can be important and I tried to tie it in in the best way possible. Let me know what you guys think of this and how it can be improved.

Also, @justinyangusa I tried to assign you as a reviewer but I guess you're not part of the org? I'll just tag you here.

Thank you friends enjoy this post!

### CODE REVIEW

File: _posts/2018-05-24-mentorship-open-source-hackillinois.markdown [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+![The hackers from the Puppeteer to Istanbul project present at closing ceremonies](/assets/article_images/mentors-open-source-hackillinois/puppeteer.jpg "puppeteer")
+
+# Sponsors
+
+Sponsors at HackIllinois had a great presence in Open Source as well! Companies had tons of opportunities to get involved with hackers on their projects. Nearly all companies brought mentors that worked with students on contributing and creating open source projects. Even companies that didn't have their own open source projects were able to work with students, because the mentors that came had experience working on large scale projects and were able to contribute alongside students. The one-on-one attention and guidance that these mentors offered was integral to many of the projects that won prizes at HackIllinois.
+
+Companies also had the option to interact with hackers in many other ways too. Some companies sponsored a prize and were able to present it to the project of their choice at the end of the event, and some companies even had personalized lounges for hackers to work in!
```
Comment (apa-123) [inline _posts/2018-05-24-mentorship-open-source-hackillinois.markdown]: I'd say this paragraph has less relevance than the previous one -- maybe mention certain prizes' focus and how they had relevance to open source?
since we don't have any company/sponsor focused blog posts however, I do see the importance of this part. Any other thoughts on how to better incorporate this? could potentially write a separate blog post or add more into this one about sponsor experience apart from open source

File: _posts/2018-05-24-mentorship-open-source-hackillinois.markdown [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+layout: post
+title: "Open Source at HackIllinois 2018"
+date: 2018-05-24 00:00:00
+categories: recap
+tags: HackIllinois2018, featured
+image: /assets/article_images/mentors-open-source-hackillinois/panel.jpg
+author: Rohin Bhasin
+---
+
+Open Source carried a huge presence at HackIllinois 2018. This year, we had over [40 mentors](https://2018.hackillinois.org/mentors) and 25 sponsors at HackIllinois, and everyone came eager to contribute to the open source community! The goal of HackIllinois has transitioned in recent years to connect with the developer community; to encourage students and hackers from all over the country to contribute to the tools they use every day. HackIllinois 2018 was a giant step towards fulfilling this vision.
```
Comment (apa-123) [inline _posts/2018-05-24-mentorship-open-source-hackillinois.markdown]: "expanded" instead of "transitioned"

Comment (apa-123) [inline _posts/2018-05-24-mentorship-open-source-hackillinois.markdown]: "to connect with the developer community by encouraging students .... "

File: _posts/2018-05-24-mentorship-open-source-hackillinois.markdown [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+![Hackers from IUPUI work with mentors from Rust in the ECEB](/assets/article_images/mentors-open-source-hackillinois/rust.jpg "rust")
+
+# Projects
+
+Tons of open source projects found themselves welcoming new contributors at HackIllinois. We had representatives from large open source projects such as [Apache Mesos](https://github.com/apache/mesos), [NPM](https://github.com/npm/npm), [Rust](https://github.com/rust-lang/rust), [Conda](https://github.com/conda/conda), [Homebrew](https://github.com/Homebrew/brew), and [Hyper](https://github.com/zeit/hyper). These were great because students and mentors could work together to contribute to some of the tools they use everyday.
+
+HackIllinois was home to lots of smaller, growing open source projects, too, like [OpenTreeMap](https://github.com/OpenTreeMap/otm-core), [NodeSchool](https://github.com/nodeschool/nodeschool.github.io) and [PrairieLearn](https://github.com/PrairieLearn/PrairieLearn). We love these open source projects because they get great opportunities for growth at HackIllinois. In fact, mentor Michael Maurizi told me that he and his team solved every issue that he had set aside for HackIllinois 2018!
```
Comment (apa-123) [inline _posts/2018-05-24-mentorship-open-source-hackillinois.markdown]: rather than "We love these" say "These open source projects provide immense opportunities for growth at HackIllinois"

File: _posts/2018-05-24-mentorship-open-source-hackillinois.markdown [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+HackIllinois did not limit students to just contributing to open source, though. Many students developed their own open source projects to give back to the community with their own new ideas. One project, [Puppeteer To Istanbul](https://github.com/istanbuljs/puppeteer-to-istanbul) even has 69 stars on Github and was mentioned in a talk at GoogleIO! And it was born right here at HackIllinois.
```
Comment (apa-123) [inline _posts/2018-05-24-mentorship-open-source-hackillinois.markdown]: I feel like the same projects get mentioned over (obviously the most successful ones) but seems repetitive especially since we are going to be doing another open source spotlight -- this post has the potential to be combined with that one as a two part "this is what open source looked like at hackillinois" and "these are projects that resulted"

File: _posts/2018-05-24-mentorship-open-source-hackillinois.markdown [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+![The hackers from the Puppeteer to Istanbul project present at closing ceremonies](/assets/article_images/mentors-open-source-hackillinois/puppeteer.jpg "puppeteer")
+
+# Sponsors
+
+Sponsors at HackIllinois had a great presence in Open Source as well! Companies had tons of opportunities to get involved with hackers on their projects. Nearly all companies brought mentors that worked with students on contributing and creating open source projects. Even companies that didn't have their own open source projects were able to work with students, because the mentors that came had experience working on large scale projects and were able to contribute alongside students. The one-on-one attention and guidance that these mentors offered was integral to many of the projects that won prizes at HackIllinois.
```
Comment (apa-123) [inline _posts/2018-05-24-mentorship-open-source-hackillinois.markdown]: no comma after students, because? not 100% sure

Comment (apa-123) [inline _posts/2018-05-24-mentorship-open-source-hackillinois.markdown]: this paragraph also emphasizes the divide between company mentors and open source mentors (as well as companies that are not open source) -- not sure if this is something we want to bring up or how to discuss around it

File: _posts/2018-05-24-mentorship-open-source-hackillinois.markdown [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+Companies also had the option to interact with hackers in many other ways too. Some companies sponsored a prize and were able to present it to the project of their choice at the end of the event, and some companies even had personalized lounges for hackers to work in!
+
+<center>&middot;&middot;&middot;</center>
+
+We're incredibly happy that open source was such a success at HackIllinois 2018, and aim to continue the contributions next year at HackIllinois 2019! In the meantime, make sure to check out [our other recent post](https://blog.hackillinois.org/recap/2018/05/19/hackillinois-2018-recap.html) by staff member Justin Yang recapping the entirety of HackIllinois 2018.
+
+Watch out for registration for HackIllinois 2019 on our website at [hackillinois.org](https://hackillinois.org)! And if you're interested in getting involved as a sponsor or mentor, be sure to reach out to us at contact@hackillinois.org.
```
Comment (apa-123) [inline _posts/2018-05-24-mentorship-open-source-hackillinois.markdown]: overall, great job! I liked the overall open source summary -- I think it could potentially be combined with Rishi's upcoming open source highlights, just had a few grammar/phrasing here and there.

File: _posts/2018-05-24-mentorship-open-source-hackillinois.markdown (hunk 2 above)
Comment (justinyangusa) [inline _posts/2018-05-24-mentorship-open-source-hackillinois.markdown]: Would it be possible to start off with a brief 1-2 sentence intro to HackIllinois and its focus on open source. Such a prelude would help to orient readers who may not be familiar with HackIllinois (e.g., new sponsors, students, mentors).

> Open Source

How do we intend to stylize "Open Source"? Capitalization or not works fine for me, but we should be consistent. I was thinking of starting a style guide so we could consolidate our terminology and provide a jumping off point for new team members.

> 25 sponsors

Similar to what was done for mentors, let's link to the sponsors.

> has transitioned

As a nit, if we consider the transition to be complete, then use the simple past tense "transitioned".

> community; to encourage

Use a colon (:) as the separator as the phrase following is not a sentence in and of itself.

File: _posts/2018-05-24-mentorship-open-source-hackillinois.markdown [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+tags: HackIllinois2018, featured
+image: /assets/article_images/mentors-open-source-hackillinois/panel.jpg
+author: Rohin Bhasin
+---
+
+Open Source carried a huge presence at HackIllinois 2018. This year, we had over [40 mentors](https://2018.hackillinois.org/mentors) and 25 sponsors at HackIllinois, and everyone came eager to contribute to the open source community! The goal of HackIllinois has transitioned in recent years to connect with the developer community; to encourage students and hackers from all over the country to contribute to the tools they use every day. HackIllinois 2018 was a giant step towards fulfilling this vision.
+
+# Mentors
+
+Open source contributions at HackIllinois would not be possible without the many amazing mentors that attended the event this year. Mentorship is essential to the HackIllinois experience; these experienced open source contributors and engineers help students get the ball rolling on contributing to open source.
```
Comment (justinyangusa) [inline _posts/2018-05-24-mentorship-open-source-hackillinois.markdown]: In the second sentence, the first clause introduces the second: consider using a colon (:) instead of a semicolon (;).

File: _posts/2018-05-24-mentorship-open-source-hackillinois.markdown [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+author: Rohin Bhasin
+---
+
+Open Source carried a huge presence at HackIllinois 2018. This year, we had over [40 mentors](https://2018.hackillinois.org/mentors) and 25 sponsors at HackIllinois, and everyone came eager to contribute to the open source community! The goal of HackIllinois has transitioned in recent years to connect with the developer community; to encourage students and hackers from all over the country to contribute to the tools they use every day. HackIllinois 2018 was a giant step towards fulfilling this vision.
+
+# Mentors
+
+Open source contributions at HackIllinois would not be possible without the many amazing mentors that attended the event this year. Mentorship is essential to the HackIllinois experience; these experienced open source contributors and engineers help students get the ball rolling on contributing to open source. 
+
+To initially introduce students to the mentors, we held panels regarding Web/Data and Systems so students could come hear about the fields they were interested in. Afterward, mentors briefly described their projects and students had a chance to discuss with mentors and join their teams.
```
Comment (justinyangusa) [inline _posts/2018-05-24-mentorship-open-source-hackillinois.markdown]: > Systems

Systems/Miscellaneous?

File: _posts/2018-05-24-mentorship-open-source-hackillinois.markdown [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+
+Afterward, hacking began! Students began learning the ropes of open source and finding issues to solve. Many mentors had tagged issues for HackIllinois beforehand, and were prepared to guide students through the initially intimidating process of understanding a codebase and improving it. 
+
+![Students start to work with mentor Andy Schwartzmeyer on the Apache Mesos project](/assets/article_images/mentors-open-source-hackillinois/powershell.jpg "mesos")
+
+Last year, hacker Trent Spice worked with the Rust team and [won the Long Con prize](https://blog.hackillinois.org/spotlight/2017/09/29/open-source-at-hackillinois-rust-cookbook.html) for HackIllinois 2017. This year, he came back with other students from IUPUI and continued to work on Rust with mentors Steve Klabnik and Ashley Williams. HackIllinois aims to build these long term relationships between students and the open source community, and we're super glad that projects like [Rust Cookbook](https://github.com/rust-lang-nursery/rust-cookbook) and [Rustbridge](https://github.com/rustbridge) are still being contributed to and built on.
```
Comment (justinyangusa) [inline _posts/2018-05-24-mentorship-open-source-hackillinois.markdown]: > IUPUI

Indiana University–Purdue University Indianapolis (IUPUI)

> long term

long-term

> Rustbridge

RustBridge

> we're super glad that projects like … are still being contributed to and built on

we're super glad that groups from HackIllinois are still working together to contribute to and built on projects like …

File: _posts/2018-05-24-mentorship-open-source-hackillinois.markdown [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+
+![Hackers from IUPUI work with mentors from Rust in the ECEB](/assets/article_images/mentors-open-source-hackillinois/rust.jpg "rust")
+
+# Projects
+
+Tons of open source projects found themselves welcoming new contributors at HackIllinois. We had representatives from large open source projects such as [Apache Mesos](https://github.com/apache/mesos), [NPM](https://github.com/npm/npm), [Rust](https://github.com/rust-lang/rust), [Conda](https://github.com/conda/conda), [Homebrew](https://github.com/Homebrew/brew), and [Hyper](https://github.com/zeit/hyper). These were great because students and mentors could work together to contribute to some of the tools they use everyday.
```
Comment (justinyangusa) [inline _posts/2018-05-24-mentorship-open-source-hackillinois.markdown]: > These were great

These mentors? These projects? Clarify.

> everyday

every day

File: _posts/2018-05-24-mentorship-open-source-hackillinois.markdown (hunk 3 above)
Comment (justinyangusa) [inline _posts/2018-05-24-mentorship-open-source-hackillinois.markdown]: > told me

Perhaps use "mentioned" or "stated" instead to keep the same tone and perspective throughout the piece.

File: _posts/2018-05-24-mentorship-open-source-hackillinois.markdown (hunk 4 above)
Comment (justinyangusa) [inline _posts/2018-05-24-mentorship-open-source-hackillinois.markdown]: > just contributing to open source

only contributing to existing open source projects

 > even has 69 stars on Github

, even has 70+ stars on GitHub as of the time of publication

> GoogleIO

Google I/O
We could also include a link to the talk or the tweet.

File: _posts/2018-05-24-mentorship-open-source-hackillinois.markdown (hunk 5 above)
Comment (justinyangusa) [inline _posts/2018-05-24-mentorship-open-source-hackillinois.markdown]: > large scale projects

large-scale projects

I think we should remove to rephrase the sentence about companies that do not have open source. I think it would distract new sponsors from the value that bringing mentors for specific projects provides.

File: _posts/2018-05-24-mentorship-open-source-hackillinois.markdown (hunk 1 above)
Comment (justinyangusa) [inline _posts/2018-05-24-mentorship-open-source-hackillinois.markdown]: > sponsored a prize and were able to present it to the project of their choice

sponsored prizes and were able to present them to the projects of their choice

File: _posts/2018-05-24-mentorship-open-source-hackillinois.markdown [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+
+Companies also had the option to interact with hackers in many other ways too. Some companies sponsored a prize and were able to present it to the project of their choice at the end of the event, and some companies even had personalized lounges for hackers to work in!
+
+<center>&middot;&middot;&middot;</center>
+
+We're incredibly happy that open source was such a success at HackIllinois 2018, and aim to continue the contributions next year at HackIllinois 2019! In the meantime, make sure to check out [our other recent post](https://blog.hackillinois.org/recap/2018/05/19/hackillinois-2018-recap.html) by staff member Justin Yang recapping the entirety of HackIllinois 2018.
```
Comment (justinyangusa) [inline _posts/2018-05-24-mentorship-open-source-hackillinois.markdown]: This paragraph would be a good place to encourage readers to continue or start contributing before HackIllinois 2019.

> https://blog.hackillinois.org/recap/2018/05/19/hackillinois-2018-recap.html

I believe there's a way to reference this post relatively.

File: _posts/2018-05-24-mentorship-open-source-hackillinois.markdown [hunk 12]
```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Open Source at HackIllinois 2018"
+date: 2018-05-24 00:00:00
+categories: recap
+tags: HackIllinois2018, featured
```
Comment (justinyangusa) [inline _posts/2018-05-24-mentorship-open-source-hackillinois.markdown]: Do we want to add `mentors` as a tag?

File: _posts/2018-05-24-mentorship-open-source-hackillinois.markdown (hunk 2 above)
Comment (apa-123) [inline _posts/2018-05-24-mentorship-open-source-hackillinois.markdown]: going off of what Justin said, I think it could be valuable to add more information about how open source gets started at HackIllinois -- mentioning that hackers have the option to create their own projects like a traditional hackathon and make it open source vs. contributing (maybe doing this without using create/contribute or making it seem like two separate tracks since this is a problem we ran into in the past)

Comment (smg155) [reply _posts/2018-05-24-mentorship-open-source-hackillinois.markdown]: I would definitely agree with what Api and Justin sare saying. I think it would valuable to add in more about how OS works at HackIllinois for those who aren't familiar. Then we also should include the fact that its not completely OS as we do want to make sure our public image includes both sides (even if it does favor OS)

File: _posts/2018-05-24-mentorship-open-source-hackillinois.markdown (hunk 10 above)
Comment (smg155) [reply _posts/2018-05-24-mentorship-open-source-hackillinois.markdown]: Consider using a more active voice in the last sentence. Just being picky, its better to stray away from passiveness (i.e. the "these were great because" part)

File: _posts/2018-05-24-mentorship-open-source-hackillinois.markdown (hunk 6 above)
Comment (smg155) [reply _posts/2018-05-24-mentorship-open-source-hackillinois.markdown]: Great job Rohin! This article really puts a spotlight to the OS theme we have going. I do definitely think it should be combined with Rishi's (you said it would be on slack - good stuff). I commented on a few things, mostly grammatical, but a couple other things as well. Also, just as a general idea for the article, maybe try to tone it down on the number of exclamation marks that you use. It would just put a more clean professional tone onto the piece. For example, you use exclamatory sentences a lot as transitions where those sentences could just as well be ended with periods. It's a little picky, but I think it could do good for the article. Overall, really great job! Excited to see this one come out.

File: _posts/2018-05-24-mentorship-open-source-hackillinois.markdown (hunk 4 above)
Comment (rohinb2) [reply _posts/2018-05-24-mentorship-open-source-hackillinois.markdown]: Yup, talked with Rishi and we're leaving it at how it is rn.

File: _posts/2018-05-24-open-source-hackillinois-2018.markdown [hunk 13]
```diff
@@ -0,0 +1,45 @@
...
+
+# Mentors
+
+Open Source contributions at HackIllinois would not be possible without the many amazing mentors that attended the event this year. Mentorship is essential to the HackIllinois experience: these experienced Open Source contributors and engineers help students get the ball rolling on contributing to Open Source. 
+
+To initially introduce students to the mentors, we held panels regarding Web/Data and Systems/Miscellaneous so students could come hear about the fields they were interested in. Afterward, mentors briefly described their projects and students had a chance to discuss with mentors and join their teams.
```
Comment (smg155) [inline _posts/2018-05-24-open-source-hackillinois-2018.markdown]: Picky but saying "initially introduce" is a little redundant

File: _posts/2018-05-24-open-source-hackillinois-2018.markdown [hunk 14]
```diff
@@ -0,0 +1,45 @@
...
+
+![Mentors with experience in Web/Data discuss at the panel in Siebel](/assets/article_images/open-source-hackillinois-2018/panel_speakers.jpg "Web/Data panel")
+
+Afterward, hacking began! Students began learning the ropes of Open Source and finding issues to solve. Many mentors had tagged issues for HackIllinois beforehand, and were prepared to guide students through the initially intimidating process of understanding a codebase and improving it.
```
Comment (smg155) [inline _posts/2018-05-24-open-source-hackillinois-2018.markdown]: you do not actually need the comma after "beforehand" since there is only one subject predicate pair in the whole sentence. I see that it could possibly be useful in breaking up the sentence however, so whatever you think would be clearest to the reader is fine here, just wanted to point it out

Comment (rohinb2) [reply _posts/2018-05-24-open-source-hackillinois-2018.markdown]: Wow big grammar boy LOL but kk

File: _posts/2018-05-24-open-source-hackillinois-2018.markdown [hunk 15]
```diff
@@ -0,0 +1,45 @@
...
+
+HackIllinois was home to lots of smaller, growing Open Source projects, too, like [OpenTreeMap](https://github.com/OpenTreeMap/otm-core), [NodeSchool](https://github.com/nodeschool/nodeschool.github.io) and [PrairieLearn](https://github.com/PrairieLearn/PrairieLearn).  These Open Source projects have immense opportunities for growth at HackIllinois. In fact, mentor Michael Maurizi mentioned that he and his team solved every issue that he had set aside for HackIllinois 2018!
+
+HackIllinois did not limit students to only contributing to existing Open Source projects, though. Many students developed their own Open Source projects to give back to the community with their own new ideas. One project, [Puppeteer to Istanbul](https://github.com/istanbuljs/puppeteer-to-istanbul), even has 70+ stars on GitHub as of the time of publication and was [mentioned in a talk at Google I/O](https://youtu.be/lhZOFUY1weo?t=19m4s). And it was born right here at HackIllinois.
```
Comment (apa-123) [inline _posts/2018-05-24-open-source-hackillinois-2018.markdown]: could maybe link more projects than just this one (similar to the discussion on slack -- don't know what the conclusion from this was) not sure how feasible this is

File: _posts/2018-05-24-open-source-hackillinois-2018.markdown [hunk 16]
```diff
@@ -0,0 +1,45 @@
...
+
+![The hackers from the Puppeteer to Istanbul project present at the closing ceremony](/assets/article_images/open-source-hackillinois-2018/puppeteer.jpg "Puppeteer to Istanbul")
+
+<center>&middot;&middot;&middot;</center><br>
+
+We're incredibly happy that Open Source was such a success at HackIllinois 2018, and aim to continue the contributions next year at HackIllinois 2019. In the meantime, make sure to check out [our other recent post]({{ site.baseurl }}{% post_url 2018-05-19-hackillinois-2018-recap %}) recapping the entirety of HackIllinois 2018.
```
Comment (apa-123) [inline _posts/2018-05-24-open-source-hackillinois-2018.markdown]: no comma after 2018

Comment (apa-123) [reply _posts/2018-05-24-open-source-hackillinois-2018.markdown]: could maybe separate into two or reword the first sentence

File: _posts/2018-05-24-open-source-hackillinois-2018.markdown [hunk 17]
```diff
@@ -0,0 +1,45 @@
...
+​	Afterward, hacking began! Students began learning the ropes of Open Source and finding issues to solve. Many mentors had tagged issues for HackIllinois beforehand and were prepared to guide students through the initially intimidating process of understanding and improving a codebase.
+
+![Students start to work with mentor Andy Schwartzmeyer on the Apache Mesos project](/assets/article_images/open-source-hackillinois-2018/powershell.jpg "mesos")
+
+Last year, hacker Trent Spice worked with the Rust team and [won the Long Con prize](https://blog.hackillinois.org/spotlight/2017/09/29/open-Source-at-hackillinois-rust-cookbook.html) for HackIllinois 2017. This year, he came back with other students from Indiana University–Purdue University Indianapolis (IUPUI) and continued to work on Rust with mentors Steve Klabnik and Ashley Williams. HackIllinois aims to build these long-term relationships between students and the Open Source community, and our staff is super glad that groups from HackIllinois are still working together to contribute to and build on projects like [Rust Cookbook](https://github.com/rust-lang-nursery/rust-cookbook) and [RustBridge](https://github.com/rustbridge).
```
Comment (redsn0w422) [inline _posts/2018-05-24-open-source-hackillinois-2018.markdown]: Link to Steve and Ashley somewhere on the Internet (either personal pages or GitHub, up to you)

File: _posts/2018-05-24-open-source-hackillinois-2018.markdown [hunk 18]
```diff
@@ -0,0 +1,45 @@
...
+
+![Hackers from IUPUI work with mentors from Rust in the ECEB](/assets/article_images/open-source-hackillinois-2018/rust.jpg "rust")
+
+We also had some awesome mentors representing Open Source projects from the fields of Data Science. We had one of the original founding members of [Julia](https://github.com/JuliaLang/julia), Jeff Bezanson, join us to show students the world of technical computing. Of course, we also had mentor Carol Willing (past Director of the Python Software Foundation) returning from HackIIllinois 2017 to work on [Jupyter](https://github.com/jupyter/jupyter).
```
Comment (redsn0w422) [inline _posts/2018-05-24-open-source-hackillinois-2018.markdown]: Link for Jeff

Comment (redsn0w422) [reply _posts/2018-05-24-open-source-hackillinois-2018.markdown]: Link for Carol

File: _posts/2018-05-24-open-source-hackillinois-2018.markdown [hunk 19]
```diff
@@ -0,0 +1,45 @@
...
+
+# Projects
+
+Tons of Open Source projects found themselves welcoming new contributors at HackIllinois. We had representatives from large Open Source projects such as [Apache Mesos](https://github.com/apache/mesos), [NPM](https://github.com/npm/npm), [Rust](https://github.com/rust-lang/rust), [Conda](https://github.com/conda/conda), [Homebrew](https://github.com/Homebrew/brew), and [Hyper](https://github.com/zeit/hyper). These influential projects gave students and mentors the opportunity to work together to contribute to some of the most common tools they use everyday.
+
+HackIllinois was home to lots of smaller, growing Open Source projects, too, like [OpenTreeMap](https://github.com/OpenTreeMap/otm-core), [NodeSchool](https://github.com/nodeschool/nodeschool.github.io) and [PrairieLearn](https://github.com/PrairieLearn/PrairieLearn).  These Open Source projects have immense opportunities for growth at HackIllinois. In fact, mentor Michael Maurizi told me that he and his team solved every issue that he had set aside for HackIllinois 2018!
```
Comment (redsn0w422) [inline _posts/2018-05-24-open-source-hackillinois-2018.markdown]: Link for Michael. Also I'd rephrase to "In fact, one of our mentors, Michael Maurizi, ..." or something similar.

File: _posts/2018-05-24-open-source-hackillinois-2018.markdown [hunk 20]
```diff
@@ -0,0 +1,45 @@
...
+
+HackIllinois did not limit students to only contributing to Open Source projects, though. Many students developed their own Open Source projects to give back to the community with their own new ideas. One project, [Puppeteer To Istanbul](https://github.com/istanbuljs/puppeteer-to-istanbul), even has 70+ stars on GitHub as of the time of publication, and was [mentioned in a talk at Google I/O.](https://youtu.be/lhZOFUY1weo?t=19m5s) And it was born right here at HackIllinois.
```
Comment (redsn0w422) [inline _posts/2018-05-24-open-source-hackillinois-2018.markdown]: I would make just "talk" the link.

File: _posts/2018-05-24-open-source-hackillinois-2018.markdown [hunk 21]
```diff
@@ -0,0 +1,45 @@
...
+
+![The hackers from the Puppeteer to Istanbul project present at closing ceremonies](/assets/article_images/open-source-hackillinois-2018/puppeteer.jpg "puppeteer")
```
Comment (redsn0w422) [inline _posts/2018-05-24-open-source-hackillinois-2018.markdown]: Change caption to "The Puppeteer to Istanbul team presenting at closing ceremonies"

File: _posts/2018-05-24-open-source-hackillinois-2018.markdown [hunk 22]
```diff
@@ -0,0 +1,45 @@
...
+
+# Mentors
+
+Open Source contributions at HackIllinois would not be possible without the many amazing mentors that attended the event this year. Mentorship is essential to the HackIllinois experience: these veteran Open Source contributors and engineers help students get the ball rolling on contributing to Open Source. 
+
+To initially introduce students to the mentors, we held panels regarding Web/Data and Systems/Miscellaneous so students could come hear about the fields they were interested in. Afterward, mentors briefly described their projects and students had a chance to discuss with mentors and join their teams.
+
+![Mentors with experience in Web/Data discuss at the panel in Siebel](/assets/article_images/open-source-hackillinois-2018/panel_speakers.jpg "panel")
+
+​	Afterward, hacking began! Students began learning the ropes of Open Source and finding issues to solve. Many mentors had tagged issues for HackIllinois beforehand and were prepared to guide students through the initially intimidating process of understanding and improving a codebase.
```
Comment (redsn0w422) [inline _posts/2018-05-24-open-source-hackillinois-2018.markdown]: Also why is there a space at the beginning of this line? Just delete it

File: _posts/2018-05-24-open-source-hackillinois-2018.markdown (hunk 18 above)
Comment (apache8080) [inline _posts/2018-05-24-open-source-hackillinois-2018.markdown]: This whole paragraph sounds very passive. This might read a little better:
"We also had some awesome mentors representing Open Source Data Science projects. Jeff Bezanson, creator of Julia, worked with hackers on solving issues for the Julia language. Carol Willing (former Director of the Python Software Foundation) joined us for another year and worked with hackers on the Jupyter Project"

Another possible rewrite:
"Jeff Bezanson, creator of Julia, worked with hackers on solving issues for the Julia language, and Carol Willing (former Director of the Python Software Foundation) joined us for another year and worked with hackers on the Jupyter Project"

File: _posts/2018-05-24-open-source-hackillinois-2018.markdown [hunk 23]
```diff
@@ -12,25 +12,25 @@ Open Source carried a huge presence at HackIllinois 2018. This year, we had over
...
 
 ![Hackers from IUPUI work with mentors from Rust in ECEB](/assets/article_images/open-source-hackillinois-2018/rust.jpg "Rust")
 
-We also had some awesome mentors representing Open Source projects from the fields of Data Science. We had one of the original founding members of [Julia](https://github.com/JuliaLang/julia), Jeff Bezanson, join us to show students the world of technical computing and had mentor Carol Willing (former Director of the Python Software Foundation) returning from HackIllinois 2017 to work on Jupyter.
+We also had some awesome mentors representing Open Source projects from the fields of Data Science. We had one of the original founding members of [Julia](https://github.com/JuliaLang/julia), Jeff Bezanson, join us to show students the world of technical computing. Of course, we also had mentor Carol Willing (former Director of the Python Software Foundation) returning from HackIIllinois 2017 to work on [Jupyter](https://github.com/jupyter/jupyter).
```
Comment (apache8080) [inline _posts/2018-05-24-open-source-hackillinois-2018.markdown]: This whole paragraph sounds very passive. This might read a little better:
"We also had some awesome mentors representing Open Source Data Science projects. Jeff Bezanson, creator of Julia, worked with hackers on solving issues for the Julia language. Carol Willing (former Director of the Python Software Foundation) joined us for another year and worked with hackers on the Jupyter Project"

Another possible rewrite:
"Jeff Bezanson, creator of Julia, worked with hackers on solving issues for the Julia language, and Carol Willing (former Director of the Python Software Foundation) joined us for another year and worked with hackers on the Jupyter Project"

**Comment by redsn0w422** [comment]
Looks good once the issues Justin mentioned are fixed.
